packages/wrangler/src/index.tsx

```diff
@@ -608,6 +609,23 @@ export function createCLIParser(argv: string[]) {
 			}
 		}
 	);
+	wrangler.command(
+		"upgrade",
+		"🆕 Upgrade Wrangler installation to latest version",
+		(yargs) => {
+			yargs
+				.option("yes", {
+					describe: 'Answer "yes" to any prompts for new projects',
```

Comment:
```suggestion
					describe: 'Answer "yes" to any prompts for upgrading',
```

Reply:
Omg... thats what I get for copy pasting and not looking